pipelines/measurement/measurement_tract.yaml

```diff
@@ -22,3 +22,29 @@ tasks:
         config.measure.minSep = 5.0
         config.measure.maxSep = 20.0
         config.measure.shearConvention = False
+  SkyObjectMean:
+    class: lsst.faro.measurement.TractTableMeasurementTask
+    config:
+      measure.statistic: MEAN
+      connections.package: pipe_analysis
+      connections.metric: SkyObjectMean_ap09Flux
+      python: |
+        from lsst.faro.measurement import FluxStatisticTask
+        import lsst.faro.utils.selectors as selectors
+        config.measure.retarget(FluxStatisticTask)
+        config.measure.columnsBand = {"flux": "ap09Flux"}
+        config.measure.selectorActions.FlagSelector=selectors.FlagSelector
+        config.measure.selectorActions.FlagSelector.selectWhenTrue=["sky_object"]
+  SkyObjectStd:
+    class: lsst.faro.measurement.TractTableMeasurementTask
+    config:
+      measure.statistic: STDEV
+      connections.package: pipe_analysis
+      connections.metric: SkyObjectStd_ap09Flux
+      python: |
+        from lsst.faro.measurement import FluxStatisticTask
+        import lsst.faro.utils.selectors as selectors
+        config.measure.retarget(FluxStatisticTask)
+        config.measure.columnsBand = {"flux": "ap09Flux"}
+        config.measure.selectorActions.FlagSelector=selectors.FlagSelector
+        config.measure.selectorActions.FlagSelector.selectWhenTrue=["sky_object"]
```

Comment:
This looks good. For posterity, I just want to note that these metric definitions should ideally be only in measurement_tract_table.yaml, and not this pipelines file, since they are using the `objectTable`. However, I recognize that this is a temporary work-around to get these metrics to appear in regular processing, since we haven't yet switched the main pipelines over to using parquet tables.